README.md

```diff
@@ -1,32 +1,94 @@
-# AWS %description% Terraform module
+# AWS Fargate Scheduled Task Terraform Module
 
-Terraform module for %long_description%
+Terraform module for scheduling Fargate tasks with CloudWatch Event Rules.
 
 ![terraform v0.11.x](https://img.shields.io/badge/terraform-v0.11.x-brightgreen.svg)
 
 ## Usage
 
 ```HCL
-resource "aws_s3_bucket" "bucket" {
-    bucket = "test"
+module "fargate-scheduled-task" {
+  source  = "baikonur-oss/fargate-scheduled-task/aws"
+  version = "0.1.4"
+
+  name                = "dev-batch-foo"
+  schedule_expression = "cron(40 1 * * ? *)"
+  is_enabled          = "true"
+
+  target_cluster_arn = "dev"
+
+  task_definition_arn = "${aws_ecs_task_definition.ecs_task_definition.arn}"
+  task_role_arn       = "${module.iam_ecs_tasks.arn}"
+  task_count          = "1"
+
+  subnets         = "subnet-***1,subnet-***2"
+  security_groups = "sg-***1,sg-***2"
```

Comment:
Same above. Change variable name from `security_groups ` to `security_group_ids` and pass a list of security group ids.